source/_docs/guides/jenkins.md

```diff
@@ -0,0 +1,223 @@
+
+## What you’ll need
+
+- Root access to a server running Jenkins. Assigning a domain name to this server, e.g. "ci.yourdomain.com," is recommended.
+- Install the following applications, both locally and on the Jenkins server. Verify the default Jenkins user has the ability to run them on the Jenkins server from the command line. 
+    - [Composer](https://getcomposer.org/) 
+    - [Drush](http://docs.drush.org/en/8.x/install-alternative)
+    - [Terminus](https://github.com/pantheon-systems/terminus#installing-with-composer)
+    - [Terminus Build Tools plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin)
+- The following Jenkins plugins should be enabled: [Git](https://wiki.jenkins-ci.org/display/JENKINS/Github+Plugin) , [Github](https://wiki.jenkins-ci.org/display/JENKINS/GitHub+Plugin), [Github Pull Request Builder](https://wiki.jenkins-ci.org/display/JENKINS/GitHub+pull+request+builder+plugin), [Environment Injector](https://wiki.jenkins-ci.org/display/JENKINS/EnvInject+Plugin), [Conditional Build Step](https://wiki.jenkins-ci.org/display/JENKINS/Conditional+BuildStep+Plugin), and [Run Condition](https://wiki.jenkins-ci.org/display/JENKINS/Run+Condition+Plugin).
+- A Drupal site on Pantheon, with [multidev](https://pantheon.io/features/multidev-cloud-environments) enabled.
+- A [GitHub](https://github.com) account.
+
+## Step 1: Local Project instantiation
+
+1. From your local terminal, use Composer to make a new local project based on our example, which contains Drupal 8, behat, and other configuration settings. In this example our project is call called "jenkins-int":
+
+```nohighlight
+SITE="my-site"
+composer create-project pantheon-systems/example-drops-8-composer $SITE
+cd $SITE
+composer prepare-for-pantheon
+```
+
+2. Initialize a local Git repository within your project. 
+
+```nohighlight
+git init
+git add -A .
+git commit -m "Initial commit."
+```
+   
+## Step 2: Create and configure a GitHub Repository
+
+1. Create a new GitHub repository, without creating a readme or .gitignore file.
+
+![New empty repository](/source/docs/assets/images/integrations/jenkins/new_repo.png)
+
+
+2. Next, connect your local project to this repo as "origin," and push the code to the master branch:
+
+```nohighlight
+git remote add origin git@github.com:YOUR-ORG/YOUR-PROJECT.git
+git push -u origin master
+```
+
+At this point, you should be able to create a local branch, commit a change, and push to GitHub. On GitHub, you should then be able to open a pull request successfully.
+
+## Step 3: Create a Pantheon Site
+
+Now we will spin up a Drupal 8 site on Pantheon with Terminus, then overwrite the default install with the code from our GitHub repo.
+
+1. From your local terminus, use terminus to create a site on Pantheon:
+```nohighlight
+terminus site:create $SITE "My Site" "Drupal 8" --org="My Team"
+terminus connection:set $SITE.dev git
+```
+
+2. Add the Pantheon remote repository address and push the code to it.
```

Comment:
I think you need a call to `composer prepare-for-pantheon` here -- but I haven't tried it. Does it work without it?